webapp/src/routes/BlockProducers.js

```diff
@@ -14,6 +15,28 @@ import ProducerSearch from '../components/ProducerSearch'
 import Tooltip from '../components/Tooltip'
 import NodeCard from '../components/NodeCard'
 import InformationCard from '../components/InformationCard'
+import { eosConfig } from '../config'
+
+const CHIPS_FILTERS = [
+  { offset: 0, where: null, limit: 28, pibot: 1 },
+  {
+    offset: 0,
+    where: { total_rewards: { _neq: 0 } },
+    limit: 21,
+    pibot: 1
+  },
+  {
+    offset: 21,
+    where: { total_rewards: { _neq: 0 } },
+    limit: 43,
```

Comment:
where does this `43` come from?

Reply:
it's the query filter to get just the paid standby without the top 21 BP

I'm not sure if we have a specific amount number for paid standby (such as top 21), but I have understood that the network has 64 paid standby BPs including the top 21.

so the limit value is the 64 BPs paid standby minus the top 21 BP.

Reply:
We get it from https://bloks.io/ because only the first 64 position have rewards (the last is cypherglasss at position 64) so we subtract the first 21 and get the 43, we dont have the rank in the backend table right now so we have to sort by votes to get the rank and from there skip the first 21 to get the paid standby

Reply:
The problem with this is that it wont always be 43. The way pay works is that if total rewards is greater than 100 EOS a day then the BP goes into a paid - standby position.

we should use the BP rewards logic based on votes to calculate this

Reply:
https://github.com/eoscostarica/eosio-dashboard/blob/d0e9bdb4d795d78cf141feaa982852ffe19efc6e/hapi/src/services/eosio.service.js#L54

this data could be helpful

Reply:
@xavier506 what about remove the `limit` and just get all the BP that have total_rewards higher than 0 ?

Reply:
yes but the minumim ammount paid to a BP is 100 EOS a day 

which means that even though a BP has votes it only gets paid if votes adds up to over 100 EOS a day

Reply:
@xavier506 right now we save `total_rewards` only if the producer have 100 EOS or higher so it's valid to use the filter by total_rewards, because in the producer table only top 21 producers and paid standby will have rewards so we can get all producers with more than 0 rewards and skip the first 21 to display only paid standby producers, that sounds good to you?

Reply:
or even better the filter could be where `total_rewards` it's higher or equal to 100 so with that we have all paid producers and then we only need to skip the first 21 (using `offset`) to display only the paid standby

Reply:
if we are already taking this into account then yes we should use rewards to determine if the node is paid / unpaid standby